content/configuration/node.md

```diff
@@ -5,9 +5,16 @@ contributors:
   - sokra
   - skipjack
   - oneforwonder
+  - Rob--W
 ---
 
-These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser. This feature is provided by webpack's internal [`NodeStuffPlugin`](https://github.com/webpack/webpack/blob/master/lib/NodeStuffPlugin.js).
+These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules.
+This allows code originally written for the Node.js environment to run in other environments like the browser.
+This feature is provided by webpack's internal
+[`NodeStuffPlugin`](https://github.com/webpack/webpack/blob/master/lib/NodeStuffPlugin.js) plugin.
+If the target is "web" (default) or "webworker", the
+[`NodeSourcePlugin`](https://github.com/webpack/webpack/blob/master/lib/node/NodeSourcePlugin.js)
+plugin is also activated.
```

Comment:
Can you leave the formatting as is? We may automatically enforce a max line length at some point (webpack-contrib/webpack-defaults#73) but for now we typically keep paragraphs all on one line and encourage contributors to word-wrap while editing these files. That said, it probably makes sense to split it into 2 paragraphs as such:

``` md
These options configure whether to polyfill or mock certain [Node.js globals](https://nodejs.org/docs/latest/api/globals.html) and modules. This allows code originally written for the Node.js environment to run in other environments like the browser. 

This feature is provided by webpack's internal [`NodeStuffPlugin`](https://github.com/webpack/webpack/blob/master/lib/NodeStuffPlugin.js) plugin. If the target is "web" (default) or "webworker", the [`NodeSourcePlugin`](https://github.com/webpack/webpack/blob/master/lib/node/NodeSourcePlugin.js) plugin is also activated.
```